### PR TITLE
Ops file to use IAM instance profile for director.

### PR DESCRIPTION
**Pull request by genevievelesperance**
https://www.pivotaltracker.com/story/show/145499709

@zachgersh

### CODE REVIEW

**Comment by cppforlife** [comment]
@genevievelesperance do we want to have two different iam profiles? one for director and one for rest?

**Comment by genevievelesperance** [comment]
@cppforlife I believe this ops-file is just for the director.. or does it impact the rest?

**Comment by cppforlife** [comment]
i think by setting default iam instance prpfile it will set it on all machines created by the director.

File: aws/iam-instance-profile.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- type: remove
+  path: /instance_groups/name=bosh/properties/aws/secret_access_key
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/aws/credentials_source?
+  value: env_or_profile
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/aws/default_iam_instance_profile?
```
Comment (evanfarrar) [inline aws/iam-instance-profile.yml]: @cppforlife @genevievelesperance so if we only wanted to give this to the director VM, then this specifically, we should not be doing, right? or is there more?

**Comment by genevievelesperance** [comment]
We can create another iam instance profile for the default_iam_instance_profile for the rest of the vms.

**Comment by genevievelesperance** [comment]
@cppforlife We are only giving the IAM instance profile to the director with this PR.

**Pull request merged**